Library/Formula/libvoikko.rb

```diff
@@ -21,5 +28,16 @@ def install
                           "--prefix=#{prefix}",
                           "--with-dictionary-path=#{HOMEBREW_PREFIX}/lib/voikko"
     system "make", "install"
+
+    resource("voikko-fi").stage {
+      ENV["PATH"] = ENV["PATH"] + ":#{bin}"
+      system "make", "vvfst"
+      system "make", "vvfst-install", "DESTDIR=#{lib}/voikko"
+      lib.install_symlink "voikko"
+    }
+  end
+
+  test do
+    pipe_output("#{Formula["libvoikko"].bin}/voikkospell -m", "onkohan\n")
```

Comment:
You can use `bin` instead of `Formula["libvoikko"].bin` because you are *in* the `libvoikko` formula here.